docs/guide/protocols/plainmd.rst

```diff
@@ -1,4 +1,55 @@
 Plain MD Protocol
 =================
 
-issue #743
\ No newline at end of file
+Overview
+--------
+
+The :class:`.PlainMDProtocol` enables the user to run an MD simulation of a ``ChemicalSystem``, which can contain e.g. a solvated protein-ligand complex, or a ligand and water.
```

Comment:
or a ligand in vacuum? - ah I see it's mentioned below, might not be necessary!

Reply:
```suggestion
The :class:`.PlainMDProtocol` enables the user to run a Molecular Dynamics (MD) simulation of a ``ChemicalSystem``, which can contain e.g. a solvated protein-ligand complex, or a ligand and water.
```

Sorry this is probably pedantic, but it's the first time you're defining it here I think?